scripts/Source/sslThreadModel.psc

```diff
@@ -1367,7 +1370,10 @@ function CenterOnObject(ObjectReference CenterOn, bool resync = true)
 		BedStatus[1] = 0
 		int Pos = Positions.Find(CenterOn as Actor)
 		if Pos >= 0
-			ActorAlias(Positions[Pos]).LockActor()
+			int SlotID = FindSlot(Positions[Pos])
```

Comment:
Just to prevent error when the position don't have ActorAlias like when the ChangeActor function is used